lib/private/Http/Client/ClientService.php

```diff
@@ -55,9 +56,17 @@ public function __construct(IConfig $config,
 	}
 
 	/**
+	 * @param CookieJarInterface $cookieJar
 	 * @return Client
 	 */
-	public function newClient(): IClient {
-		return new Client($this->config, $this->certificateManager, new GuzzleClient());
+	public function newClient(?CookieJarInterface $cookieJar): IClient {
```

Comment:
```suggestion
	public function newClient(?CookieJarInterface $cookieJar = null): IClient {
```

This is needed so we don't break old calls without any arguments